exercises/random_number.cpp

```diff
@@ -4,3 +4,15 @@
   Output:
   The random number is: 4
 */
+
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+using namespace std;
+
+int main(){
+  srand(time(0));
+
+  cout << "The random number is: " << rand() << endl;
+}
```

Comment:
manca return 0, inoltre potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```